sorc/fvcom_tools.fd/process_FVCOM.f90

```diff
@@ -49,7 +50,7 @@ program process_FVCOM
 
 !  Grid variables
 
-   character*180 :: geofile
+   character*180 :: thisfv3file
    character*2 :: workPath
```

Comment:
I can't seem to find why 'geofile' was changed?

Reply:
goefile is a local variable for file names. It is used by several kinds of the restart files. I think the name just to avoid confusion by its name.